lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -141,10 +141,27 @@ def sync_bash_command_on_dockerfile(repo_name, dockerfile_build_path, build_cmd)
         sync_bash_command(repo_name, build_cmd, file_path, branch_name, sync_path, commit_id_path)
       end
 
+      def check_and_fix_repo(repo_path)
```

Comment:
@dobbymoodge Are you saying bash doesn't have enough capability to run the conditionals?